litestar/typing.py

```diff
@@ -318,12 +318,12 @@ def is_required(self) -> bool:
     @property
     def is_annotated(self) -> bool:
         """Check if the field type is Annotated."""
-        return Annotated in self.type_wrappers  # type: ignore[comparison-overlap]
+        return bool(self.metadata)
 
     @property
     def is_literal(self) -> bool:
         """Check if the field type is Literal."""
-        return get_origin(self.annotation) is Literal
+        return self.origin is Literal
```

Comment:
Similar to above, this isn't related to the PR, but a minor improvement to previous.